### DISCUSSION TITLE
Video loaded Black & White - Can't fix on my own - Please help!!!

### DISCUSSION THREAD

**Discussion by phuongpham2501** [SubtitleEdit/subtitleedit#6395] [Q&A]
I don't know why caused this. I have deleted and re-installed, used portable version and all didn't work.
![image](https://user-images.githubusercontent.com/44307097/200824700-7d31179d-2eed-4731-a325-9dc1d31b506a.png)

**Comment by niksedk** [comment]
Using DirectShow video player? Do you have the latest https://github.com/Nevcairiel/LAVFilters/releases ? (use installer exe)

Alternatively, you could switch video player to mpv.

**Reply by phuongpham2501** [reply]
Hi Mr. Niksedk,

I have fixed this problem via download K-lite Codec -> Fresh Install.

Thanks for your support <3
